lib/internal/socketaddress.js

```diff
@@ -142,6 +142,12 @@ class InternalSocketAddress extends JSTransferable {
   constructor(handle) {
     super();
     this[kHandle] = handle;
+    this[kDetail] = this[kHandle]?.detail({
+      address: undefined,
+      port: undefined,
+      family: undefined,
+      flowlabel: undefined,
+    });
```

Comment:
What bug is being fixed here?

Reply:
The inner detail was not actually being set by the `InternalSocketAddress` causing it to throw if you actually used it.